main.cpp

```diff
@@ -19,13 +19,22 @@ using namespace clang::tooling;
 
 class CastCallBack : public MatchFinder::MatchCallback {
 public:
-    CastCallBack(Rewriter& rewriter) {
+    CastCallBack(Rewriter& rewriter):rwrtr(rewriter) {
         // Your code goes here
     };
 
     void run(const MatchFinder::MatchResult &Result) override {
         // Your code goes here
+	const auto* castExprPtr = Result.Nodes.getNodeAs<clang::CStyleCastExpr>("cast");

```

Comment:
What if `castExprPtr` is a null pointer? How to protect the program from crashing in such cases?

Reply:
if ( castExprPtr == nullptr )
{ return ; }